stdlib/parser.mc

```diff
@@ -8,23 +8,45 @@ let debug = lam s. if debug_flag then printLn s else ()
 -- The Parser monad -----------------------------
 
 type Pos = (String, Int, Int)
```

Comment:
Could this perhaps be changed to use a record instead?

Reply:
That makes sense! If I recall correctly, this was written before there were records in the language 🙂

Reply:
Fixed now! Thanks!